### DISCUSSION TITLE
Adding a new team member

### DISCUSSION THREAD

**Discussion by miguel-a-calles-mba** [serverless-heaven/serverless-webpack#1548] [Team Posts]
https://github.com/j0k3r has been pretty involved in commenting and reviewing PRs. Any thoughts on adding him to the team?

**Comment by HyperBrain** [comment]
Just let me know about your decision and I will add him.
BTW: You're doing a good job!

**Comment by miguel-a-calles-mba** [comment]
Thanks!

I asked him if he's interested.

**Comment by miguel-a-calles-mba** [comment]
He said he's interested in joining the team. I suggest adding him if no one disagrees.

**Comment by miguel-a-calles-mba** [comment]
@j0k3r is interested in joining the team. Are we good to add him? Thanks.

**Comment by HyperBrain** [comment]
Ok. Then I will add him. Looks like no one disagreed,

**Comment by HyperBrain** [comment]
@miguel-a-calles-mba I just invited him to the team :)

**Comment by miguel-a-calles-mba** [comment]
Thanks!

Welcome to the team, @j0k3r.
